google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineEnvironment.java

```diff
@@ -23,7 +23,8 @@
  */
 public enum AppEngineEnvironment {
   APP_ENGINE_STANDARD("appengine.environment.name.standard"),
-  APP_ENGINE_FLEX("appengine.environment.name.flexible");
+  APP_ENGINE_FLEX("appengine.environment.name.flexible"),
+  APP_ENGINE_FLEX_COMPAT("appengine.environment.name.flexible");
```

Comment:
I feel like the only time we've discussed this is when you've suggested this should be changed to something else. :)

This is one of those places where I feel like we could go on in circles forever. Wouldn't it be simple to just consider that APP_ENGINE_FLEX_COMPAT should be considered as FLEXIBLE, for remote purposes, and STANDARD for internal IDE stuff? This is my understanding of the FLEX_COMPAT environment, and I was able to live with it very well while building this PR.